<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>menu</class>
 <widget class="QDialog" name="menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>261</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>SELECT</string>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QRadioButton" name="bRadioButton">
     <property name="text">
      <string>BORDER</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="textStyleRadioButton">
     <property name="text">
      <string>TEXT STYLE</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="textShadowRadioButton">
     <property name="text">
      <string>TEXT SHADOW</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="fomtgRadioButton">
     <property name="text">
      <string>OTHER FORMATTING OPTIONS</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Go---&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;---Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="launchButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Launch the web page</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
